src/components/AddBooks.scss

```diff
@@ -1,6 +1,91 @@
-.add-books {
-  width: 90%;
+.wrapper {
+  width: 75rem;
   margin: 0 auto;
+}
+
+.wrapper h1 {
```

Comment:
Well done, implementing the suggestions provided by previous reviewer.
I have observed that you mistakenly miss one of the suggestion:
![image](https://user-images.githubusercontent.com/61361037/230473741-588d2d79-001e-4bd5-80f1-f6c180bdaf9b.png)

This was the comment:
![image](https://user-images.githubusercontent.com/61361037/230473917-c96b3a97-1253-4a35-a519-601dc9203eeb.png)

Reply:
Thanks for reminding me of that honestly I did forget, I have fixed it now, thanks once again.